### DISCUSSION TITLE
Roccat Vulcan 2 Max support

### DISCUSSION THREAD

**Discussion by Adam-LX** [X3n0m0rph59/eruption#231] [Hardware Support]
Hi there,

I have recently bought the roccat vulcan 2 max keyboard, and found your project here. It seems impressive, although my device isn't supported yet. I would like to ask you if there is a chance to support vulcan 2 devices as well? I'm a begginer but I would gladly help with some detailed info if it'll be needed.

**Comment by X3n0m0rph59** [comment]
Hi @Adam-LX, thank you for your hardware support request!

Development on a driver for the `ROCCAT Vulcan 2 Max` will commence soon. I can't tell how long it will take until I have a working prototype ready, but it is probably a few months away still.

**Comment by Tr3b0s** [comment]
I have a Roccat Vulcan 2 without the max. And I wanted to ask if the devices are so similar that there is no need to open another Hardware Support discussion. If you need additional information, I will do the best I can to provide them. And if it is clear that they aren't similar, I will open a new Discussion.

**Reply by Adam-LX** [reply]
It's been a while since I last posted here, and the commits looking like there were just for maintanance purposes. Is the project still alive?

**Comment by takosalad** [comment]
I tried to add the Vulcan II Max to duncanthrax' code, but it doesn't work.
Anyway, device is Bus 001 Device 014: ID 1e7d:2ee2 ROCCAT ROCCAT Vulcan II Max
